example/00-etcd-config.yaml

```diff
@@ -0,0 +1,17 @@
+# It can be used to run etcd and etcd-backup-restore locally for testing purpose.
+
+# Human-readable config for single member etcd.
+name: etcd
+data-dir: "default.etcd"
+metrics: extensive
+snapshot-count: 75000
+enable-v2: false
+quota-backend-bytes: 1073741824 # 1Gi
```

Comment:
Why not use 8GB itself? It would be helpful for local testing for large DB sizes.

Reply:
done